### ISSUE TITLE
query result grid becomes unreachable using keyboard after switching views

### ISSUE THREAD

**Issue by alanrenmsft** [microsoft/azuredatastudio#17898]
1. run a query that returns results
2. now user can use keyboard navigation to reach the grid
2. switch to messages tab
3. switch back to results tab 
4. now user cannot reach the grid using keyboard navigation

![before-table-keyboard](https://user-images.githubusercontent.com/13777222/145686548-eec1d0be-954d-4d09-b9ea-028f2bed2743.gif)

**Comment by alanrenmsft** [comment]
fixed.
